src/sgame/sg_bot_parse.cpp

```diff
@@ -332,6 +337,11 @@ static AIValue_t numOurBuildings( gentity_t* self, const AIValue_t *params )
 	return AIBoxInt( count );
 }
 
+static AIValue_t numOurSpawns( gentity_t* self, const AIValue_t* )
+{
+	return AIBoxInt( level.team[ G_Team( self ) ].numSpawns );
```

Comment:
`numSpawns` updates on spawn (of the spawning structure) and death so the semantics are in fact identical to `numOurBuildings` FWIW.